mvp.feature

```diff
@@ -1171,27 +1214,23 @@ Feature: MVP features for NVA
 
   # Menuitems for Administrator
   @359
-  Scenario: Administrator opens User Administration
+  Scenario: Administrator opens User Administration (Part I)
```

Comment:
I find the Part I, Part II to be a bit problematic. I think the problem is that the titles describe the initial action of the user and not the desired functionality. 

suggestion:

```
Scenario Outline: Administrator views Sections with Role-members
    Given that the user is logged in as Administrator
    When they click the menu item User Administration
    Then they see the User Administration page
    And they see the <Section>
    And they see the <Section> contains a list of all users affiliated with their institution and with with role <Role>
    And they see under each Section a Button to assign the <Role> to a another user
    And they see that each list has the field
      | Authentication ID |
    And they see a button Remove that is enabled for each user
    Examples:
      | Section                    | Role          |
      | Institution Administrators | Administrator |
      | Curators                   | Curator       |
      | Editors                    | Editor        |
      | Creators                   | Creator       |
```

Reply:
replaced with this description